docs/Developers/Review.rst

```diff
@@ -117,6 +117,17 @@ For the Reviewers
    behaviour well. E.g. for bear tests just testing for a good and bad file
    is **not** sufficient.
 
+.. note::
+
+    While reviewing pull requests or patches for ``difficulty/low`` issues,
+    make sure that the patch solves the issue and doesn't create any
+    further issues.
```

Comment:
I believe the current language lead more to "Be _absolutely certain_ that there are no further issues" instead of inferring "There should be no _obvious consequences_ of the patch". IMO, the former one might be discouraging to new reviewers. I think the the latter one should be preferred - Along with a tone of "Don't review if you are not aware about working of the code yourself" (which is already included in the text later). Still, your call.

Reply:
It is for the `difficulty/low` reviewers, so it needs to be accurate as they may apply for the developer promotion and lots of the time false review creates a mess on PR  and generally happens while reviewing `difficulty/low` PR, and the Tone of `Don't review` is demanded by maintainer and I also think the tone needs to be there and I have taken care of `discouraging` part, that's why I didn't place it on newcomer page, I placed it on `Review` page so that It don't effect directly to any newcomer when they read `Newcomer Guide`, after that they maybe review `Newcomer` PR,  when it comes to `difficulty/low` PR, the tone `needs to be there` to remind them.

Reply:
`Coala` is absolutely `newcomer friendy` no doubt but when It come to review part , IMO using the `direct tone` is much better than removing the `mess`

Reply:
I agree with you both in a way, I think direct tone is good, but restricting reviews' demands too much might "scare" people away^^

Reply:
But it's merged, and I think this is still a good start for further

Reply:
> I agree with you both in a way, I think direct tone is good, but restricting reviews' demands too much might "scare" people away

Yeah, that's exactly my point - We would require something to point to "You needn't be an expert to do a review, you just need to be responsible enough to know by yourself if you understand the working of code enough" as I said later in the comment below.

Reply:
I of-course wasn't sure if that's exactly what ```coala``` would want to convey (so, I didn't ```unack```ed it) :D